P5/Source/Specs/noteGrp.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+    Copyright TEI Consortium.
+    Dual-licensed under CC-by and BSD2 licences
+    See the file COPYING.txt for details
+    $Date$
+    $Id$
+--><?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?><elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="core" ident="noteGrp">
+  <desc versionDate="2005-01-14" xml:lang="en">contains a group of notes</desc>
+  <classes>
+    <memberOf key="att.global"/>
+    <memberOf key="model.noteLike"/>
+    <memberOf key="att.placement"/>
+    <memberOf key="att.pointing"/>
+    <memberOf key="att.typed"/>
+    <memberOf key="att.written"/>
+    <memberOf key="att.anchoring"/>
+    <memberOf key="model.correspDescPart"/>
+    <memberOf key="model.correspActionPart"/>
+    <memberOf key="model.correspContextPart"/>
+  </classes>
+  <content>
+    <sequence>
+        <elementRef key="desc" minOccurs="0" maxOccurs="unbounded"/>
+        <alternate minOccurs="1" maxOccurs="unbounded">
+          <elementRef key="note"/>
+          <elementRef key="noteGrp"/>
+        </alternate>
+    </sequence>
+  </content>
+  <exemplum xml:lang="la">
+    <p>In the following example, there are two notes in different languages, each specifying the content of 
+      the annotation relating to the same fragment of text:</p>
+    <egXML xmlns="http://www.tei-c.org/ns/Examples">
+      <p>... Parwum vadit theoloneum, tamen reuerendos dominos archiepiscopum et canonicos Leopolienses
```

Comment:
Incredibly pedantic comment; feel free to ignore: "Parwum" might be what the source has, but I'm suspicious, because the correct Latin form would be "Paruum" or "Parvum" (if your orthography distinguishes 'u' and 'v'—"reuerendos" doesn't, since it's not "reverendos"). Latin doesn't have a 'w', though all sorts of weirdness can happen in later Latin :-).

Reply:
To satisfy your sweet Latin tooth, I'm no palaeographer ;-) 

![image](https://user-images.githubusercontent.com/449468/106354820-abdf1c00-62f4-11eb-9276-bf680d77acb2.png)

Reply:
There is another case in the same document they seem to be using w (3rd line gwerrarum). I'm happy to shorten the example though, to avoid controversy

![image](https://user-images.githubusercontent.com/449468/106354936-5eaf7a00-62f5-11eb-875f-bbfe97f53a87.png)

Reply:
I think it is "parvum". But I agree that's "gwerrarum" in the other example, :-)

I am not a paleographer either. What is this text?